core/const.cpp

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "const.hpp"
+
+#define DEFINE(x) decltype(x) x
+
+namespace fc {
+  DEFINE(kConsensusMinerMinPower){BigInt{1} << 40};
```

Comment:
need macro for this?

Reply:
With this macro you don't need to repeat type.